bpf/kprobe_pwru.c

```diff
@@ -316,7 +333,7 @@ handle_everything(struct sk_buff *skb, struct pt_regs *ctx, bool has_get_func_ip
   SEC(PWRU_KPROBE_TYPE "/skb-" #X)                                             \
   int kprobe_skb_##X(struct pt_regs *ctx) {                                    \
     struct sk_buff *skb = (struct sk_buff *) PT_REGS_PARM##X(ctx);             \
-    return handle_everything(skb, ctx, PWRU_HAS_GET_FUNC_IP);                  \
+    return kprobe_skb(skb, ctx, PWRU_HAS_GET_FUNC_IP);                         \
```

Comment:
Nit: empty space at the end.

Reply:
The spaces are used for "\"'s alignment.

And there's no space at EOL.